WooCommerce/Classes/POS/Presentation/Reusable Views/POSModalManager.swift

```diff
@@ -2,18 +2,33 @@ import SwiftUI
 
 class POSModalManager: ObservableObject {
     @Published private(set) var isPresented: Bool = false
+    @Published var allowsInteractiveDismissal: Bool = true
```

Comment:
I think this can be `private(set)`:
```suggestion
    @Published private(set) var allowsInteractiveDismissal: Bool = true
```

Reply:
5cfb3d6
Good catch, I used it in a binding briefly and forgot to change it back when that didn't work.